Automattic/wp-calypso
# Remove site style step from flows and update default theme to Maywood

#### Changes proposed in this Pull Request

* Update default theme for `blog`, `business`, and `professional` segments to use `pub/maywood`
* Remove `site-style-with-preview` step from flows
* Remove site style step from e2e tests
* Remove the site style import to reduce bundle size, and instead use a hard-coded font URL
* Include the Gutenberg `theme.css` file, which includes overrides for separators
* Add a couple of styling overrides to ensure site preview matches newly created sites using the Maywood theme

#### Testing instructions



* Apply D32188-code and D32317-code
* Complete signup flows from `http://calypso.localhost:3000/start` and ensure that the preview looks like the screenshots below. Test on Chrome, Firefox, Safari, and IE11
* Ensure that all flows where the `site-style-with-preview` step has been removed are still functioning correctly (`business`, `premium`, `personal`, `free`, and `onboarding` flows)
* Ensure that the created site is using the Maywood theme
* Create an e-commerce site, and ensure that it is still using the `pub/dara` theme

Screenshots of the site preview and newly created sites for each of the affected site types (segments):

## Blog preview versus newly created site



![image](https://user-images.githubusercontent.com/14988353/63992539-b4e02280-cb2f-11e9-8bcd-3446ba31a4b7.png)

## Business preview versus newly created site



![maywood-business](https://user-images.githubusercontent.com/14988353/63992712-92023e00-cb30-11e9-8fb2-952ce1e13ace.png)

## Professional preview versus newly created site

![maywood-professional](https://user-images.githubusercontent.com/14988353/63992863-4c924080-cb31-11e9-9dd1-589d072c35e4.png)

Related to issue #35842
